woningwaardering/stelsels/utils.py

```diff
@@ -720,7 +720,7 @@ def classificeer_ruimte(ruimte: EenhedenRuimte) -> Ruimtesoort | None:
         Ruimtedetailsoort.berging.code,
         Ruimtedetailsoort.wasruimte.code,
         Ruimtedetailsoort.kelder.code,
-        # Ruimtedetailsoort.schuur.code,
+        Ruimtedetailsoort.schuur.code,
     ] or (
         Ruimtedetailsoort.schuur.naam == ruimte.detail_soort.naam
     ):  # Schacht en schuur hebben dezelfde code
```

Comment:
```suggestion
        Ruimtedetailsoort.schuur.code,
    ] or (
        Ruimtedetailsoort.schuur.naam == ruimte.detail_soort.naam
    ):  # Schacht en schuur hebben dezelfde code
```
volgensmij hebben wij schuur uitgecomment omdat `# Schacht en schuur hebben dezelfde code`, dus ik weet niet of we het zomaar kunnen uncommenten?

Reply:
Oh crap, dus eigenlijk hadden we dit al geimplementeerd 😄

Reply:
haha ja blijkbaar wel :)